src/core/project/qgsprojectstylesettings.h

```diff
@@ -144,6 +145,67 @@ class CORE_EXPORT QgsProjectStyleSettings : public QObject
      */
     QgsStyle *projectStyle();
 
+    /**
+     * Set project \a colorModel
+     *
+     * It would serve as default color model when selecting a color in the whole application.
+     * Any color defined in a different color model than the one specified here will be converted to
+     * this color model when exporting a layout.
+     *
+     * If a color space has already been set and its color model differs from \a colorModel, project
+     * color space is set to invalid one. \see setColorSpace() colorSpace()
+     *
+     * defaults to Qgis::ColorModel::Rgb
+     *
```

Comment:
Do we need some documentation here about the different behavior on qt5 vs qt 6 builds?

Reply:
I added the following 

> (This is only true if QGIS is built against Qt 6.8.0 or greater)